pages/03.guides/05.start-reading/comic-manga-reader/default.en.md

```diff
@@ -32,7 +32,7 @@ When in the Web Reader click of tap in the center of the screen to bring up the
 ! Note: All Menu options scale with the screen size of your device display.
 
 #####**Image Splitting Options**: Right to Left or Left to Right
-- Splitting is the act of taking a spread page (a page that contains both left and right as one image) and virtually splits it so it feels like 2 pages. This is useful if you don't want to scroll while reading on a phone or tablet. Right to Left will split the image so the right side of the image renders first, then the left part as the second page. This is the most common.
+- Splitting is the act of taking a spread page (a page that contains both left and right as one image) and virtually splitting it so it feels like 2 pages. This is useful if you don't want to scroll while reading on a phone or tablet. Right to Left will split the image so the right side of the image renders first, then the left part as the second page. This is the most common.
```

Comment:
Can we add if you don't want to scroll or zoom on phone or tablet, since we now have Fit to Screen which fits everything in, but you usually have to zoom to read the text.